### DISCUSSION TITLE
Image Slider slide expiration date

### DISCUSSION THREAD

**Discussion by ghost** [PrestaShop/PrestaShop#35510] [Ideas]
With this little feature we can prevent to show expired slider banners on homepage.

https://github.com/PrestaShop/ps_imageslider/pull/39

![expiration](https://user-images.githubusercontent.com/8243273/54746722-30c35f80-4bcd-11e9-903b-45c8732449e8.png)

![expiration2](https://user-images.githubusercontent.com/8243273/54746771-49cc1080-4bcd-11e9-87c7-9f1d1f733ac2.png)

**Comment by khouloudbelguith** [comment]
Hi @PrestaShark,

Thanks for your suggestion & for your PR fix.
@marionf, @colinegin what do you think?
Thanks!

**Comment by colinegin** [comment]
Hello @PrestaShark, thanks for your idea and PR !
It's a nice improvement, I think we can review your code and test it.

We'll keep you posted.

Have a nice day !

**Comment by Quetzacoalt91** [comment]
Does the UX team want to have a look at this?

**Comment by TristanLDD** [comment]
Hi @PrestaShark, thank you for your contribution. 

I suggest a few modifications on your interface: 

- The expiration date would be better under the name of the slide, so it's position will not depend on the length of the name, so dates will be easier to 'scan'
- The color indication is not inclusive enough as some people might have troubles to differentiate colors. It's always better not to use color as the only information vector and add a clear wording. 
- Please use the font open-sans for the date ;) 

That would give us something like that: 
![Capture d’écran 2019-04-23 à 17 54 59](https://user-images.githubusercontent.com/27484094/56596509-fa746800-65f0-11e9-8275-e3c7ef332cfc.png)

**Comment by ghost** [comment]
Hi guys and sorry for delay. I will work with all mentioned suggestions and brb soon! Have a good day and keep going with 1.7.6 :)

**Comment by hibatallahAouadni** [comment]
Hello @PrestaShark 

Could you please apply the changes requested by @jolelievre on your PR.

Thanks!

**Comment by hibatallahAouadni** [comment]
Hello @PrestaShark ,

Since we had no news from you for more than 20 days, I'll close this ticket to avoid cluttering up the backlog.
Please note that you can always create a new one if further information pops up.

Thanks!

**Comment by hibatallahAouadni** [comment]
Ping @alfonsolr09

**Comment by alfonsolr09** [comment]
https://github.com/PrestaShop/ps_imageslider/pull/59
@hibatallahAouadni
